src/renderers/shared/fiber/ReactFiberCommitWork.js

```diff
@@ -53,10 +60,22 @@ module.exports = function<T, P, I, TI, PI, C, CX, PL>(
     getPublicInstance,
   } = config;
 
+  function callComponentWillUnmountWithTimerInDev(current, instance) {
```

Comment:
It should be because Uglify will find no references to it. At least I think we've been doing this in many places and it worked before.